datapipelines/src/main/kotlin/io/customer/sdk/DataPipelineInstance.kt

```diff
@@ -1,9 +1,69 @@
 package io.customer.sdk
 
 import io.customer.sdk.android.CustomerIOInstance
+import io.customer.sdk.data.model.CustomAttributes
+import kotlinx.serialization.SerializationStrategy
 
 /**
  * Extends [CustomerIOInstance] to provide the instance of CustomerIO SDK with
  * Data Pipelines implementation.
  */
-interface DataPipelineInstance : CustomerIOInstance
+interface DataPipelineInstance : CustomerIOInstance {
+    /**
+     * Custom profile attributes to be added to current profile.
+     */
+    var profileAttributes: CustomAttributes
+
+    /**
+     * Identify a customer (aka: Add or update a profile).
+     * [Learn more](https://customer.io/docs/identifying-people/) about identifying a customer in Customer.io
+     * Note: You can only identify 1 profile at a time in your SDK. If you call this function multiple times,
+     * the previously identified profile will be removed. Only the latest identified customer is persisted.
+     *
+     * @param userId ID you want to assign to the customer.
+     * This value can be an internal ID that your system uses or an email address.
+     * [Learn more](https://customer.io/docs/api/#operation/identify)
+     */
+    fun identify(userId: String)
```

Comment:
To keep our SDK compatible with Java, we have to add two separate methods to the interface as using `JvmOverloads` and default values isn't possible with interface at the moment.

Reply:
I think we don't have to stick to interface, lets just utilize abstract class and add `JvmOverloads`